lib/src/detectors/hyperv_detector.cc

```diff
@@ -0,0 +1,24 @@
+#include <internal/detectors/hyperv_detector.hpp>
+#include <internal/vm.hpp>
+
+using namespace std;
+
+namespace whereami { namespace detectors {
+
+   result hyperv(const sources::cpuid_base& cpuid_source, sources::smbios_base& dmi_source)
+   {
+       result res {vm::hyperv};
+
+       if (cpuid_source.vendor() == "Microsoft Hv") {
```

Comment:
Hah, also applies to Windows Virtual PC!

https://en.wikipedia.org/wiki/CPUID#EAX.3D0:_Get_vendor_ID_.28including_EAX.3D1:_Get_CPUID.29